docs/src/modules/components/AppNavDrawer.js

```diff
@@ -172,22 +177,83 @@ function AppNavDrawer(props) {
               </Box>
             </NextLink>
             {process.env.LIB_VERSION ? (
-              <Button
-                component="a"
-                href={`https://mui.com${languagePrefix}/versions/`}
```

Comment:
A back story on why the current version. It's designed so that users browsing the docs of v4 can click on the link and see the docs of v5 linked. From what I understand of the current approach, once we d
put v6 out, users on v5 won't see v6 linked. Is this important? Not sure, but at least, to consider.

Reply:
> Update the main README.md to have the v4 visible/searchable without having to click a button/accordion.

That's definitely a good thing to do. It can be done outside of this PR I guess but it would for sure help with the pain I'm tackling here.

> From what I understand of the current approach, once we'd put v6 out, users on v5 won't see v6 linked

I guess not?! If v6 was already out, people on v5 would see v6 as the first item of the menu instead of v5 being the first as it is now. The first item should always be the latest version.

Reply:
> If v6 was already out, people on v5 would see v6 as the first item of the menu instead of v5 being the first as it is now.

I was considering what would happen for developers browsing https://v5.mui.com/ they won't see v6 in the dropdown, it could confuse them. But it's a marginal downside, the new tradeoff is great.